lab/lab2/js/part3-application.js

```diff
@@ -42,6 +44,22 @@ var getAndParseData = function() {
   /* =====================
     Fill out this function definition
   ===================== */
+  dat = $.ajax("https://raw.githubusercontent.com/CPLN692-MUSA611/datasets/master/json/philadelphia-crime-snippet.json";).done(
```

Comment:
It looks like you are copying a dataset from a previous years class. 

Furthermore, this does not successfully run for me. 

Does it for you?